### PR TITLE
credentialsAreValid in backend, no security breach

### PR DESCRIPTION
**Pull request by amiaoh**
localhost response should now only show 'true/false' upon submit

### CODE REVIEW

File: app/Login.tsx [hunk 1]
```diff
@@ -1,29 +1,23 @@
...
-    return;
-  }
+//   if (elementWithMatchingUsername === undefined) {
+//     return;
+//   }
 
-  return elementWithMatchingUsername.password === inputPassword;
-}
+//   return elementWithMatchingUsername.password === inputPassword;
+// }
```
Comment (yungyhan) [inline app/Login.tsx]: is this still needed?

File: server/getCredentials.ts [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
+  const validLoginDetails: Credentials[] = [
+    {
+      username: "FakeUser1",
+      password: "Password1",
+    },
+    {
+      username: "FakeUser2",
+      password: "Password2",
+    },
+  ];
```
Comment (yungyhan) [inline server/getCredentials.ts]: any reason for this const to live inside the function? doing so means it's recreated every time the function runs but doesn't look like it changes

File: server/getCredentials.ts [hunk 3]
```diff
@@ -1,17 +1,26 @@
...
+  const elementWithMatchingUsername = validLoginDetails.find((loginDetails) => {
+    return loginDetails.username === inputUsername;
+  });
+  if (!elementWithMatchingUsername) {
+    return;
+  }
+  return elementWithMatchingUsername.password === inputPassword;
```
Comment (yungyhan) [inline server/getCredentials.ts]: there's a different array method that would do this more concisely

**Review by yungyhan — CHANGES_REQUESTED** [review]
overall structure and logic looks reasonable, some improvements as per comments

File: app/Login.tsx (hunk 1 above)
Comment (amiaoh) [reply app/Login.tsx]: nope, removed 
https://github.com/auberginearms/starterApp/pull/1/commits/db49e2a731d23e491395235c649b063cc23da78d

File: server/getCredentials.ts (hunk 2 above)
Comment (amiaoh) [reply server/getCredentials.ts]: okay moved it outside
https://github.com/auberginearms/starterApp/pull/1/commits/db49e2a731d23e491395235c649b063cc23da78d

File: server/getCredentials.ts (hunk 3 above)
Comment (amiaoh) [reply server/getCredentials.ts]: https://github.com/auberginearms/starterApp/pull/1/commits/db49e2a731d23e491395235c649b063cc23da78d

**Review by yungyhan — APPROVED** [review]
don't put all that other junk in next time

**Pull request merged**